### PR TITLE
feat: add missing 'context-menu' values

### PR DESCRIPTION
**Pull request by codebytere**
#### Description of Change

This PR fills in a nontrivial amount of data missing from the object emitted by the 'context-menu' event. It'd been a significant amount of time since this was last updated, so a number of fields have been added to the object which may be useful to users handling the event and which we should therefore expose.

cc @zcbenz @jkleinsc @MarshallOfSound 

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] `npm test` passes
- [x] relevant documentation is changed or added
- [x] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes

Notes: Added missing fields to the parameters emitted as part of the `context-menu` event.

### CODE REVIEW

File: docs/api/web-contents.md [hunk 1]
```diff
@@ -674,6 +686,8 @@ Returns:
     * `canPaste` Boolean - Whether the renderer believes it can paste.
     * `canDelete` Boolean - Whether the renderer believes it can delete.
     * `canSelectAll` Boolean - Whether the renderer believes it can select all.
+    * `canTranslate` Boolean - Whether the renderer believes it can perform translation.
+    * `canEditRichly` Boolean - Whether the renderer believes it can edit text richly.
```
Comment (jkleinsc) [inline docs/api/web-contents.md:691]: `canSave` is missing from docs here

**Review by jkleinsc — CHANGES_REQUESTED** [review]
LGTM apart from a small doc change needed

File: docs/api/web-contents.md [hunk 2]
```diff
@@ -664,16 +672,23 @@ Returns:
...
     is able to perform the corresponding action.
     * `canUndo` Boolean - Whether the renderer believes it can undo.
     * `canRedo` Boolean - Whether the renderer believes it can redo.
     * `canCut` Boolean - Whether the renderer believes it can cut.
-    * `canCopy` Boolean - Whether the renderer believes it can copy
+    * `canCopy` Boolean - Whether the renderer believes it can copy.
     * `canPaste` Boolean - Whether the renderer believes it can paste.
     * `canDelete` Boolean - Whether the renderer believes it can delete.
     * `canSelectAll` Boolean - Whether the renderer believes it can select all.
+    * `canTranslate` Boolean - Whether the renderer believes it can perform translation.
```
Comment (MarshallOfSound) [inline docs/api/web-contents.md:690]: This will never be `true` in Electron as we don't (and won't) have a translation service

**Review by MarshallOfSound — CHANGES_REQUESTED** [review]
Can we make sure this is tagged as `feat:`

**Review by MarshallOfSound — APPROVED** [review]

**Review by zcbenz — APPROVED** [review]

**Review by jkleinsc — APPROVED** [review]

**Comment by jkleinsc** [comment]
API LGTM

**Review by nornagon — COMMENTED** [review]
There are a few fields that still aren't exposed here, e.g. [writing_direction_*](https://source.chromium.org/chromium/chromium/src/+/master:content/public/common/untrustworthy_context_menu_params.h;l=127;drc=02f28360f2f6588ca2428ef93c8129e0a8cc231c), [impression](https://source.chromium.org/chromium/chromium/src/+/master:content/public/common/untrustworthy_context_menu_params.h;l=75;drc=02f28360f2f6588ca2428ef93c8129e0a8cc231c). Are there any particular reasons for these omissions?

**Review by miniak — APPROVED** [review]

**Comment by sick78** [comment]
so what i not get paid ?

**Comment by sick78** [comment]
so what i not get paid ?

**Pull request merged**